examples/Docker/kafka/prodkafka.py

```diff
@@ -0,0 +1,28 @@
+from time import sleep
+from json import dumps
+from kafka import KafkaProducer
+
+producer = KafkaProducer(bootstrap_servers=['0.0.0.0:9092'],
+                         value_serializer=lambda x: 
+                         dumps(x).encode('utf-8'))
+
+count = 0
+while(True):
+    data = {
```

Comment:
This looks like something specific to your use case. Doge is data producer why we would need this script?

Reply:
Yes, no need to for this script, will be removed